internal/internal_schedule_client.go

```diff
@@ -199,6 +199,7 @@ func (sc *scheduleClient) List(ctx context.Context, options ScheduleListOptions)
 			Namespace:       sc.workflowClient.namespace,
 			MaximumPageSize: int32(options.PageSize),
 			NextPageToken:   nextToken,
+			Query:           options.Query,
```

Comment:
Is there any issue on the server side when passing both a `Query` string and a `NextPageToken`? I don't think we do this anywhere else in the Go SDK and want to make sure the server does the correct thing.

Reply:
ListWorkflowExecutions? The SDK doesn't call it but it exposes it to be used that way. IOW the token does not encapsulate the query, you still need the query.

Reply:
yeah the SDK doesn't wrap `ListWorkflowExecutions ` so I wasn't sure on the contract.